ci/container/Dockerfile

```diff
@@ -186,9 +186,6 @@ RUN cargo install cargo-audit --version ${CARGO_AUDIT_VERSION}
 # Add zshrc generated from zsh-newuser-install (option 2)
 COPY --chown=ubuntu:ubuntu ./ci/container/files/zshrc /home/ubuntu/.zshrc
 
-# Whitelist some SSH keys
-COPY --chmod=0700 ./ci/container/files/known_hosts /etc/ssh/ssh_known_hosts
```

Comment:
I assume we needed this before such that we could `git clone` from GitHub over SSH. Why don't we need this anymore?